common/source/docs/common-cuav-pixhawkv6X.rst

```diff
@@ -0,0 +1,179 @@
+
+- :ref:`SERIAL6_PROTOCOL<SERIAL6_PROTOCOL>` should be set to "23".
+
+- FPort would require :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` be set to "15".
+
+- CRSF would require :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` be set to "0".
+
+- SRXL2 would require :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` be set to "4" and connects only the TX pin.
+
+Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM. See :ref:`common-rc-systems` for details.
+
+PWM Output
+==========
+
+The Pixhawkv6X supports up to 16 PWM outputs. All 16 outputs
+support all normal PWM output formats. All FMU outputs, except 7 and 8, also support DShot.
+
+The 8 FMU PWM outputs are in 4 groups:
+
+ - Outputs 1, 2, 3 and 4 in group1
+ - Outputs 5 and 6 in group2
+ - Outputs 7 and 8 in group3
+
+
+FMU outputs within the same group need to use the same output rate and protocol. If
+any output in a group uses DShot then all channels in that group need
+to use DShot.
+
+Battery Monitoring
+==================
+
+The autopilot defaults are setup for CAN Power Module use (normally supplied with autopilot) and attached to one or both of its 2 dedicated power monitor ports with a pin connectors:
+
+.. image:: ../../../images/cuav-can-pmu.jpg
+   :target: ../../_images/cuav-can-pmu.jpg
+   :width: 250px
+
+- :ref:`BATT_MONITOR<BATT_MONITOR>` = 8
+- :ref:`CAN_P1_DRIVER<CAN_P1_DRIVER>` = 1
+- :ref:`CAN_P2_DRIVER<CAN_P2_DRIVER>` = 1
+- :ref:`CAN_D1_PROTOCOL<CAN_D1_PROTOCOL>` = 1
+- :ref:`CAN_D2_PROTOCOL<CAN_D2_PROTOCOL>` = 1
+
+Compass
+=======
+
+The Pixhawkv6X has a built-in compass. Due to potential
+interference, the autopilot is usually used with an external I2C compass as
+part of a GPS/Compass combination.
+
+GPIOs
+=====
+
+The 8 FMU outputs can be used as GPIOs (relays, buttons, RPM etc). To use them you need to set the output's ``SERVOx_FUNCTION`` to -1. See :ref:`common-gpios` page for more information.
+
+The numbering of the GPIOs for PIN variables in ArduPilot is:
+
+FMU pins:
+ - PWM1 50
+ - PWM2 51
+ - PWM3 52
+ - PWM4 53
+ - PWM5 54
+ - PWM6 55
+ - PWM7 56
+ - PWM8 57
+
+Additional GPIOs:
+
+ - FMU_CAP1 58
+ - NFC_GPIO 59
+
+
+Analog inputs
+=============
+
+The Pixhawkv6X has 2 analog inputs, one 6V tolerant and one 3.3V tolerant
+
+ - ADC Pin12 -> ADC 6.6V Sense
+ - ADC Pin13 -> ADC 3.3V Sense
+ - Analog 3.3V RSSI input pin = 103
+
+Connectors
+==========
+
+Unless noted otherwise all connectors are JST GH
+
+Loading Firmware
+================
+
+The board comes pre-installed with an ArduPilot compatible bootloader,
+allowing the loading of xxxxxx.apj firmware files with any ArduPilot
+compatible ground station.
+
+Firmware for these boards can be found `here <https://firmware.ardupilot.org>`_ in  sub-folders labeled "Pixhawk6X".
+
```

Comment:
A hint can be added to the cuav pixhawk 6X and pixhawk v6x documentation: Pixhawk 6x folder for CUAV Pixhawk v6x and Holybro Pixhawk 6x?

Reply:
I do not understand...this is very direct and non confusing as it is...why the additional information about a different OEM? a marketing tactic?

Reply:
This is not a marketing ploy, the pixhawk 6x is made by holybro and the pixahwk v6x is made by CUAV; since the folder names are non-neutral, I'd like to be able to add this hint to the documentation.

Reply:
I think what is in this PR is reasonable, and we don't want to specifically call out the Holybro board on this CUAV page. We will soon have several other vendors of compatible boards that use the same firmware and it would not work well to have to mention all of them